src/pykeen/experiments/cli.py

```diff
@@ -50,6 +50,11 @@ def _make_dir(_ctx, _param, value):
     is_flag=True,
     help="Discard trained models after training.",
 )
+extra_config_option = click.option(
+    "--extra-config",
+    type=pathlib.Path,
+    default=None,
```

Comment:
can you add a `help` argument with an explanantion why someone would use this

Reply:
I give the long explanation here, and try to boil that down for the help string:

The reproduction experiment configurations are part of pykeen, i.e., usually live in `site-packages/pykeen`. Moreover, they do not specify any result tracker, since they have to work for all users on all machines, and thus having none is a good default. However, some of them take a long time to run, and I may be interested in intermediate results. Now I have two options:

1. edit the experiment configurations in pykeen. This will lead to conflicts when I update `pykeen` and/or checkout another branch (if it is installed in editable mode).
2. have a (global) extra config, which, e.g., contains my login information for wandb, the MLFlow URI, etc.

This option allows me to have the best of both worlds: a shared, pykeen-owned experiment configuration file, and a custom, user-specific result tracking configuration.

Reply:
thanks :)